svc/pkg/linode/util/src/lib.rs

```diff
@@ -16,8 +16,7 @@ pub struct Client {
 }
 
 impl Client {
-	pub async fn new() -> GlobalResult<Self> {
-		let api_token = util::env::read_secret(&["linode", "token"]).await?;
+	pub async fn new(api_token: &str) -> GlobalResult<Self> {
```

Comment:
We're writing a lot of conditionals. Can we just pass in "override api token" as an optional and if not exists, read the default token?